WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.kt

```diff
@@ -186,28 +198,6 @@ class MeFragment : Fragment(R.layout.me_fragment), OnScrollToTopListener {
             }
         }
 
-        if (unifiedAboutFeatureConfig.isEnabled()) {
-            aboutTheAppContainer.isVisible = true
-
-            if (BuildConfig.IS_JETPACK_APP) {
-                meAboutIcon.setImageResource(R.drawable.ic_jetpack_logo_white_24dp)
-            }
```

Comment:
We shouldn't remove this to keep JP icon. Otherwise, the WP icon is shown on JP.
<img src="https://user-images.githubusercontent.com/2471769/209139956-297962ad-7728-475f-a667-72200012a902.png" height=640>

Reply:
Good catch 👍